### ISSUE TITLE
what does this extension do?

### ISSUE THREAD

**Issue by exekutive** [dmzx/Topic-Image#1]
Does it work for phpBB 3.3.10?

**Comment by FredislandCocci10** [comment]
Yes, but only for attachment. And each forum MUST have at least one message to prevent error message about undefined variables. You can see example of the working extension here : http://cocci10.fredisland.net/fofo/viewforum.php?f=50-yeti-sports
(phpBB 3.3.10 + PHP 8.1.2)
